proto/dydxprotocol/revshare/params.proto

```diff
@@ -19,3 +20,13 @@ message MarketMapperRevenueShareParams {
   // the revenue share goes down to 0
   uint32 valid_days = 3;
 }
+
+// MarketRevShareDetailsParams represents params for the 
+// MsgSetMarketMapperRevShareDetailsForMarket message
+message MarketRevShareDetailsParams {
```

Comment:
I would either get rid of this proto completely and just add the market_id to the top level struct or rename this to `SetMarketRevShareDetailsParams`

Reply:
done!